app/src/main/res/layout/activity_reminder.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
why not Constraintlayout? or even better try changing this to compose.

Reply:
This is something i told Ali to avoid. If they're creating xml UI for testing, keep it with separate local branch. don't push it to remote branch.